embassy-stm32/gen.py

```diff
@@ -227,6 +234,15 @@
                 }}
 
                 {''.join(irq_declares)}
+
+                pub mod exti {{
```

Comment:
- Could these go in the toplevel pac module like the rest of impls? Or is there something special about them?
- I'd keep everything that's not calling macros in the non-generated .rs files, such as these `use` statements. Makes it easier to hack on the macro'd code without having to regenerate.

Reply:
I've moved them at the toplevel pac, but I'm not sure how to get rid of the use Interrupt and InterruptExt, because they are needed for steal().